### DISCUSSION TITLE
Resolving Online Validator Errors in Price Transparency MRF Files: Guidance on Handling MS-DRG Data and Associated Scenarios

### DISCUSSION THREAD

**Discussion by siddesai1005** [CMSgov/hospital-price-transparency#86] [Q&A] [answered]
We are encountering errors when uploading price transparency MRF file to the online validator with the error message "If an item or service is encoded, a corresponding valid value must be encoded for at least one of the following: "Gross Charge", "Discounted Cash Price", "Payer-Specific Negotiated Charge: Dollar Amount", "Payer-Specific Negotiated Charge: Percentage", "Payer-Specific Negotiated Charge: Algorithm".  We have identified two scenarios that trigger these errors:
1.	Facilities Not Performing the MS-DRG Service: In cases where a particular facility does not perform the MS-DRG service, the online validator throws an error. Should we only list the DRG codes that are associated with services provided by the facility?
2.	Facilities with No Standard Charges Due to No Patients: For facilities where no patients have undergone the specific DRG, resulting in a lack of standard charges being derived, the online validator also generates an error. Is it recommended to retain or delete the associated MS-DRG line item within our file despite the error?
We kindly request your guidance on the appropriate data retention approach for these scenarios, along with any suggestions for addressing the online validator errors. Should we retain or delete the MS-DRG data in each case?

**Comment by vduffel99** [comment]
I am having this issue as well. It started a couple of days ago. I am getting this for the Min/ Max as well as other areas. All the fields have the indicator that the field can be blank so I am unsure why I continue to receive the error.

**Reply by CarlsonErin** [reply]
@vduffel99 can you provide via email to PriceTransparencyHospitalCharges@cms.hhs.gov an excerpt example from your file as we cannot repeat this error.

**Reply by CarlsonErin** [reply, marked as answer]
@siddesai1005 please review the definition of payer-specific standard charge and the FAQs on the CMS  Hospital Price Transparency Resources Page:  https://www.cms.gov/priorities/key-initiatives/hospital-price-transparency/resources. CMS defines hospital “items and services” to mean all items and services, including individual items and services and service packages, that could be provided by a hospital to a patient in connection with an inpatient admission or an outpatient department visit for which the hospital has established a standard charge. In other words, hospitals must post the standard charge (as applicable) for each item/service for which the hospital has established a standard charge.    The payer-specific negotiated charge (the charge that a hospital **has negotiated**
with a third-party payer for an item or service).  Payer-specific standard charges can be found in the hospital payer contracts and rate sheets.
